### PR TITLE
Modify the sample of pipeline step and add  description for "restApiKeyId"

### PR DESCRIPTION
**Pull request by daipresents**
The sample of "Scripted Pipeline" doesn't work so I modified it.
And my customer are confusing "restApiKeyId. They input REAL KEY not id. So I add more description for the parameter in case.

- [] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch! => No, I don't find the branch so use main branch
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,11 +106,9 @@ mabl applicationId: 'APP-ID-a', continueOnMablError: true, continueOnPlanFailure
 
 ``` syntaxhighlighter-pre
 node {
-   stage('mabl') {
-       steps {
-           step([$class: 'MablStepBuilder', restApiKeyId: 'REST-API-KEY-ID', environmentId: 'ENV-ID-e', applicationId: 'APP-ID-a'])
-       }
-   }
+  stage('mabl') {
```
Comment (twistedpair) [inline README.md:109]: Are we sure this is the correct format? Looking at the [Jenkins Pipeline docs](https://www.jenkins.io/doc/book/pipeline/), all the examples I see have `stage > stages > steps > step`. Are we sure it is OK to remove the `steps` block?

```
stages {
        stage('Build') { 
            steps { 
                sh 'make' 
            }
        }
    }
```

From the Jenkins [docs](https://www.jenkins.io/doc/book/pipeline/syntax/#stage)
> The stage directive goes in the stages section and should contain a steps section

**Review by twistedpair — COMMENTED** [review]
Thanks, @daipresents .

I have a question about `steps`.

File: README.md (hunk 1 above)
Comment (daipresents) [reply README.md:109]: @twistedpair Thank you for your comment. My customer and I tried to use your sample but we faced "java.lang.NoSuchMethodError: No such DSL method 'steps' found among steps". So DSL issue or the example is wrong.

Please check this link: https://stackoverflow.com/questions/42113655/no-such-dsl-method-stages

This example does not use "steps" for scripted pipeline too.

Comment (daipresents) [reply README.md:109]: One more information.

You can see this page for scripted pipeline(old format): https://www.jenkins.io/doc/book/pipeline/syntax/#scripted-steps

We can see this part.

>As discussed at the start of this chapter, the most fundamental part of a Pipeline is the "step". Fundamentally, steps tell Jenkins what to do and serve as the basic building block for both Declarative and Scripted Pipeline syntax.
>Scripted Pipeline does not introduce any steps which are specific to its syntax; Pipeline Steps reference contains a comprehensive list of steps provided by Pipeline and plugins.

**Review by twistedpair — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ pipeline step for mabl. This can be written by hand or created via the
 -   continueOnPlanFailure: Continue if your plan/tests fail
 -   environmentId: Selects the environment to run deployments against
     (one of environmentId or applicationId is required)
--   restApiKeyId: The id of the API key secret of the desired deployment workspace -
```
Comment (bertold) [inline README.md:88]: ID is the term that Jenkins is using in the user interface.
I would phrase this differently:
restApiKeyId: The id of the API key secret of the desired deployment workspace. The id is the value that is assigned in the Jenkins configuration to the corresponding secret. It is not the secret itself.

File: README.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ pipeline step for mabl. This can be written by hand or created via the
...
+-   restApiKeyId: The id of the API key secret of the desired deployment workspace. "restApiKeyId" is not real key -
```
Comment (bertold) [inline README.md:88]:
```suggestion
-   restApiKeyId: The id of the API key secret of the desired deployment workspace. The id is the value that is assigned in the Jenkins configuration to the corresponding secret. It is not the secret itself.
```

**Pull request merged**